### PR TITLE
feat(docs) prefix info in migration docs

### PR DESCRIPTION
**Pull request by timrombergjakobsson**
**Describe pull-request**  
A draft on how we could inform about the prefix change.

**Solving issue**  
Fixes: [DTS-1764](https://tegel.atlassian.net/jira/software/projects/DTS/boards/1?assignee=63ff6d6c314f5088138187f0&selectedIssue=DTS-1764)

**How to test**  
1. Go to Storybook
2. Check in "Migrating from components v4" document.  
3. Look at the newly added info in the document

[DTS-1764]: https://tegel.atlassian.net/browse/DTS-1764?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: core/src/stories/Migration/Migration.stories.mdx [hunk 1]
```diff
@@ -20,9 +18,18 @@ Previously we have not had a set naming convention for our events. This has led
 components where some events has had a `custom` prefix, and others have had other ways of indicating whether
 or not it is a custom event. We have now settled on a naming convention for all events.
 Both **external**, events we expect you as a user to interact with, and **internal**, events that are used within
-our components to update state/react to changes.
+our components to update state/react to changes. 
 
-In short:
+## Prefix change
```
Comment (mJarsater) [inline core/src/stories/Migration/Migration.stories.mdx:23]: Nice! Could we add the same sort of examples but for component names?

File: core/src/stories/Migration/Migration.stories.mdx [hunk 2]
```diff
@@ -20,9 +18,18 @@ Previously we have not had a set naming convention for our events. This has led
...
+Furthermore we decided to rename the previously used 'sdds'-prefix to 'tds'. 
+This occurred as a result of a deliberate choice to facilitate the transition to the new Tegel package and prevent conflicts with previously used components.
+
+In short, before the prefix change:
```
Comment (alexandereneroth) [inline core/src/stories/Migration/Migration.stories.mdx:27]:
```suggestion
For example, before the prefix change:
```
I think this works better, because this is just a few examples, not a short description of all the changes.

File: core/src/stories/Migration/Migration.stories.mdx [hunk 3]
```diff
@@ -284,26 +288,18 @@ If the component is used on a background with a color of grey-50, mode-variant p
 
 ### Updated `<sdds-inline-tabs>` and `<sdds-navigation-tabs>` and introduced `<sdds-folder-tabs>`. Introduced child components for all Tabs.
 
-Previously we had the `<sdds-inline-tabs>` and `<sdds-inline-tabs-fullbleed>`. These
-reminded little of eachother in appearence and therefore it made more sense to treat
-these as two totally seperate component. To have a clear distiction between them we
-have now renamed the `<ssd-inline-tabs>` to `<sdds-folder-tabs>`, to futher highlight
-its look and its function.
+Previously we had the `<sdds-inline-tabs>` and `<sdds-inline-tabs-fullbleed>`. These had little resemblance to each other in appearance, thus it was more logical to consider them as completely distinct components. 
+In order to establish a clear differentiation between them, we have recently renamed the `<ssd-inline-tabs>` as `<sdds-folder-tabs>` to better emphasize both its visual appearance and its functionality.
```
Comment (alexandereneroth) [inline core/src/stories/Migration/Migration.stories.mdx:292]:
```suggestion
In order to establish a clear differentiation between them, we have recently renamed the `<ssd-inline-tabs>` as `<sdds-folder-tabs>` to better reflect its visual appearance.
```
I think the use of "folder" in the name has no connection to the functionality so I'd leave that out.

File: core/src/stories/Migration/Migration.stories.mdx [hunk 4]
```diff
@@ -20,9 +18,18 @@ Previously we have not had a set naming convention for our events. This has led
...
 
-In short:
+## Prefix change
+Furthermore we decided to rename the previously used 'sdds'-prefix to 'tds'. 
+This occurred as a result of a deliberate choice to facilitate the transition to the new Tegel package and prevent conflicts with previously used components.
+
+In short, before the prefix change:
+
+- External events are named: 'sdds' + native event name. ('sddsClick', 'sddsChange')
+- Internal events are named: 'internalsdds' + event name. ('internalsddsTablePropsChange', 'internalsddsCheckboxChange')
```
[the remark this answers is not in the document]
Comment (timrombergjakobsson) [reply core/src/stories/Migration/Migration.stories.mdx:30]: How do you mean a "item"?

File: core/src/stories/Migration/Migration.stories.mdx (hunk 3 above)
Comment (timrombergjakobsson) [reply core/src/stories/Migration/Migration.stories.mdx:292]: Also, your comment highlighted the fact that it was misspelled also, <ssd-inline-tabs> is not correct :).

File: core/src/stories/Migration/Migration.stories.mdx (hunk 1 above)
Comment (timrombergjakobsson) [reply core/src/stories/Migration/Migration.stories.mdx:23]: How do you mean?

File: core/src/stories/Migration/Migration.stories.mdx (hunk 4 above)
[the remark this answers is not in the document]
Comment (alexandereneroth) [reply core/src/stories/Migration/Migration.stories.mdx:30]: an item in the list, like
- Web-component names are named with sdds- + component name (...)

File: core/src/stories/Migration/Migration.stories.mdx (hunk 1 above)
Comment (timrombergjakobsson) [reply core/src/stories/Migration/Migration.stories.mdx:23]: Or maybe you mean what Alex mentioned below?

**Comment by mJarsater** [comment]
I think this needs rebase?

File: core/src/stories/Migration/Migration.stories.mdx [hunk 5]
```diff
@@ -20,12 +18,23 @@ Previously we have not had a set naming convention for our events. This has led
...
+- Web-component names are named with sdds- + component name. ('SddsBadge')
+
+Now, after prefix change:
 
 - External events are named: 'tds' + native event name. ('tdsClick', 'tdsChange')
 - Internal events are named: 'internaltds' + event name. ('internaltdsTablePropsChange', 'internaltdsCheckboxChange')
+- Web-component names are named with tds- + component name ('TdsBadge')
```
Comment (mJarsater) [inline core/src/stories/Migration/Migration.stories.mdx:37]: This needs to be 'tds-badge'

File: core/src/stories/Migration/Migration.stories.mdx [hunk 6]
```diff
@@ -20,12 +18,23 @@ Previously we have not had a set naming convention for our events. This has led
...
-In short:
+## Prefix change
+Furthermore we decided to rename the previously used 'sdds'-prefix to 'tds'. 
+This occurred as a result of a deliberate choice to facilitate the transition to the new Tegel package and prevent conflicts with previously used components.
+
+In short, before the prefix change:
+
+- External events are named: 'sdds' + native event name. ('sddsClick', 'sddsChange')
+- Internal events are named: 'internalsdds' + event name. ('internalsddsTablePropsChange', 'internalsddsCheckboxChange')
+- Web-component names are named with sdds- + component name. ('SddsBadge')
```
Comment (mJarsater) [inline core/src/stories/Migration/Migration.stories.mdx:31]:
```suggestion
- External events are named: 'sdds' + native event name. ('sddsClick', 'sddsChange')
- Internal events are named: 'internalsdds' + event name. ('internalsddsTablePropsChange', 'internalsddsCheckboxChange')
- Web component names are named with sdds- + component name. ('SddsBadge')
```

File: core/src/stories/Migration/Migration.stories.mdx (hunk 5 above)
Comment (mJarsater) [inline core/src/stories/Migration/Migration.stories.mdx:37]:
```suggestion
- Web component names are named with tds- + component name ('TdsBadge')
```

File: core/src/stories/Migration/Migration.stories.mdx [hunk 7]
```diff
@@ -20,10 +18,21 @@ Previously we have not had a set naming convention for our events. This has led
...
+
+- Web component names are named with 'sdds-' + component name. ('sdds-badge')
+- External events are named: 'sdds' + native event name. ('sddsClick', 'sddsChange')
+- Internal events are named: 'internalsdds' + event name. ('internalsddsTablePropsChange', 'internalsddsCheckboxChange')
+
+Now, after prefix change:
+
+- Web component names are named with 'tds-' + component name ('tds-badge')
 - External events are named: 'tds' + native event name. ('tdsClick', 'tdsChange')
 - Internal events are named: 'internaltds' + event name. ('internaltdsTablePropsChange', 'internaltdsCheckboxChange')
```
Comment (mJarsater) [inline core/src/stories/Migration/Migration.stories.mdx:38]: I know it's not part of this PR. But do you want to update the link below that has a TODOLINKTOCODE href.

**Review by mJarsater — APPROVED** [review]
left one small comment on a link that's not really part of this PR. Otherwise LGTM! 🥫 <- awesome sauce

File: core/src/stories/Migration/Migration.stories.mdx (hunk 7 above)
Comment (timrombergjakobsson) [reply core/src/stories/Migration/Migration.stories.mdx:38]: Sure I can fix it! Although, think I will close this PR and open a new one. Too many merge clonflicts. @mJarsater

Comment (mJarsater) [reply core/src/stories/Migration/Migration.stories.mdx:38]: Sure thing!

Comment (timrombergjakobsson) [reply core/src/stories/Migration/Migration.stories.mdx:38]: Btw, where do we wanna link to? Here https://github.com/scania-digital-design-system/tegel/blob/main/.github/CODE_STYLE.md

Comment (mJarsater) [reply core/src/stories/Migration/Migration.stories.mdx:38]: Yeah that makes sense. May the sentence should be tweked a bit.  Read more about or naming conventions in our code convetions or something?

Comment (timrombergjakobsson) [reply core/src/stories/Migration/Migration.stories.mdx:38]: Sounds good! Ill figure something out.

**Comment by timrombergjakobsson** [comment]
Closing this one due to a lot of merge conflicts and opening a new one.
